utils.js

```diff
@@ -4398,11 +4398,45 @@ export const parseCsProjAssetsData = async function (csProjData) {
         ).toString()
       )
     }
-
   const purlString = decodeURIComponent(rootPkg["bom-ref"].toString());
   pkgList.push(rootPkg);
   let rootPkgDeps = new Set();
 
+  // create root pkg deps
+  if (csProjData.targets && csProjData.project.frameworks) {
+    const versionRegex = /[\d.]+/g;
+    for (const frameworkTarget in csProjData.project.frameworks) {
+      for (const dependencyName in csProjData.project.frameworks[frameworkTarget].dependencies) {
+
+        const versionsRaw = csProjData.project.frameworks[frameworkTarget].dependencies[dependencyName].version
+        const versionsList = versionsRaw.match(versionRegex)
+
+        for (const dependencyVersion of versionsList) {
+          const pkgNodeName = `${dependencyName}/${dependencyVersion}`
+
+          if (csProjData.targets[frameworkTarget][pkgNodeName]) {
```

Comment:
@noqcks The frameworks list in csProjData.project.frameworks is unlike the frameworks as displayed in targets, so a TypeError is thrown at this line. This is for the [Castle Core](https://github.com/castleproject/Core) repo.

Reply:
Can you show me an example snippet from the repo? I was using test data at `test/data/project.assets.json`

```
  "targets": {
    "net5.0": {
    ...
    }
  },
 "project": {
    "frameworks": {
       "net5.0": {
       ...
       }
     }
  }
```